### ISSUE TITLE
Failed to reproduce the Anymal experiment results

### ISSUE THREAD

**Issue by kevinqyh0827** [leggedrobotics/robotic_world_model#19]
Hi, thanks for contributing this great work and it is amazing!

I tried to reproduce the Anymal-D task and could not get the Fig. 5 results as mentioned in the original RWM paper.

My reproduced plot is like:
(This plot is I  train MBPO-PPO from scratch as described from the original paper, and the reward is much worse from my side.)
<img width="1321" height="533" alt="Image" src="https://github.com/user-attachments/assets/b1194d9e-c742-4770-82de-47980d47726c" />

(This plot is I "finetune" the PPO policy during the MBPO-PPO training)
<img width="1493" height="546" alt="Image" src="https://github.com/user-attachments/assets/c8924008-da90-41c5-897e-3ddbf4bd9982" />

("real sim" here means the simulator itself and there is no data from real hardware)

The reproduced 250M PPO process is as expected but the MBPO-PPO has much lower reward. I am wondering if this is because the "6M steps pre-training dynamics model data"? Since I cannot find it public on internet, I just follow the README and ran the "dynamics model pre-raining". I found that this process is training another unused PPO policy and update world model using the transition data. Is my understanding correctly? 

Besides, I noticed that the default training hyper parameters in this repo is not the same as the original paper. Which one is better? Also the default MBPO-PPO training in the README is "finetune" not "from scratch". I am wondering which one should I use to reproduce the results? 

Thanks so much!

**Comment by kevinqyh0827** [comment]
Hi, I would like to add more results here. This time I first run the dynamics model pretraining using both the default configs from this repo and configs from original paper, followed by the README file. Then I use both "finetune" and "from scratch" to train the MBPO-PPO policy. Here is the reward curve:

<img width="1131" height="379" alt="Image" src="https://github.com/user-attachments/assets/41715c99-45e3-45e2-86bc-a0aa2678f5f9" />

<img width="1296" height="311" alt="Image" src="https://github.com/user-attachments/assets/db98774b-65e1-46b6-bb18-6cce324cbbcf" />

Based on the table above, I found that the reward curve is below PPO because ~40% of episodes ending with "fall down" during the MBPO-PPO training stage. I am wondering where I am doing wrong and how to handle this problem? 

Thanks so much!
